pkg/spec/gpyrpc/gpyrpc.proto

```diff
@@ -153,7 +155,8 @@ message ExecProgram_Args {
 
 	repeated string k_filename_list = 2;
 	repeated string k_code_list = 3;
-	
+
```

Comment:
Avoid unnecessary diffs.

Reply:
these diffs are intended to keep consistent code format